drivers/abp2/abp2.c

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_abp2
+ * @{
+ *
+ * @file
+ * @brief       Honeywell ABP2 series pressure and temperature sensor driver
+ *
+ * @author      David Picard <david.picard@clermont.in2p3.fr>
+ * @}
+ */
+
+#define MAX_LOOPS_TIMEOUT   (10)            /**< Timeout (ms). */
+
+
```

Comment:
GitHub Actions / static-tests

too many consecutive empty lines

Reply:
Remove blank lines, group includes and defines